views/helpers.templ

```diff
@@ -1,48 +1,11 @@
 package views
 
-import (
-	"github.com/ugent-library/biblio-backoffice/ctx"
-	h "html"
-	"mvdan.cc/xurls/v2"
-	"strings"
-)
-
-var reURL, _ = xurls.StrictMatchingScheme("https")
+import "github.com/ugent-library/biblio-backoffice/ctx"
 
 templ CSRFTag(c *ctx.Ctx) {
 	<input type="hidden" name={ c.CSRFName } value={ c.CSRFToken }/>
 }
 
-func Linkify(text string) string {
-	text = h.EscapeString(text)
-
-	matches := reURL.FindAllStringIndex(text, -1)
-
-	b := strings.Builder{}
-	pos := 0
-	for _, match := range matches {
-		before := text[pos:match[0]]
-		if len(before) > 0 {
-			b.WriteString(before)
-		}
-
-		link := text[match[0]:match[1]]
-		b.WriteString(`<a href="`)
-		b.WriteString(link)
-		b.WriteString(`" target="_blank">`)
-		b.WriteString(link)
-		b.WriteString(`</a>`)
-		pos = match[1]
-	}
-
-	after := text[pos:]
-	if len(after) > 0 {
-		b.WriteString(after)
-	}
-
-	return b.String()
-}
-
```

Comment:
Moved this method to views/util along with StripHTML (that came from views/aria), see above. Two templ's still remain in this file (CSRFTag and Cat). Not sure if we should leave this like that, or give them their dedicated files.